api/src/main/java/jakarta/persistence/query/ReadQueryOptions.java

```diff
@@ -34,7 +34,7 @@
  * <li>a method with a {@link StaticQuery} or Jakarta Data
  *     {@code jakarta.data.repository.Query} annotation
  *     whose {@code value} member specifies a {@code SELECT}
- *     statement,</li>
+ *     statement,
```

Comment:
Entirely unrelated to the change 🙂 
But just in case... since we are now building with JDK 25 we could start using markdown for the javadoc 🫣 🙂

Reply:
Sure but we're not going to do that piecemeal. It's all or nothing.